superset/examples/birth_names.py

```diff
@@ -424,6 +424,16 @@ def create_slices(tbl: SqlaTable) -> tuple[list[Slice], list[Slice]]:
                 viz_type="table",
                 metrics=metrics,
             ),
+            query_context=get_slice_json(
```

Comment:
@michael-s-molina even though you were able to run the Celery tests locally without issue, after a few tweaks (increasing timeouts to address the performance degradation on M1 silica) I was able to see the following error:

<img width="491" alt="Screenshot 2023-07-14 at 8 10 46 PM" src="https://github.com/apache/superset/assets/4567245/b04cb0d2-0dd1-4d75-9b89-ccf6983de48c">

I don't fully grok where/when the query context is persisted (the only place I could see where the query context was generated from the payload/form-data was via the [/api/chart/v1/<pk>/data/](https://github.com/apache/superset/blob/0328dd270467e71260bfa85078beb7b38a87877b/superset/charts/data/api.py#L69-L172) RESTful API GET endpoint), but it does seem the existence of query context seemed somewhat sporadic. Maybe @villebro can provide more insights in terms of how/where this is generated/persisted as in the example data currently only the "Pivot Table v2" chart (per the `slices.query_context` column) has the query context explicitly defined yet there are swaths of non-legacy charts.

The TL;DR is similar to the "Pivot Table v2" chart when I explicitly added the query context for the "Daily Totals" chart (which backs the annotation layer in the Cypress test which was failing) the test passed.